pkg/cmd/openshift-service-serving-cert-signer/apis/serviceservingcertsigner/v1alpha1/types.go

```diff
@@ -0,0 +1,34 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	webconsolev1 "github.com/openshift/api/webconsole/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type ServiceServingCertSignerConfig struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// ServingInfo is the HTTP serving information for the controller's endpoints
+	ServingInfo webconsolev1.HTTPServingInfo `json:"servingInfo"`
```

Comment:
@openshift/api-review let's talk about api helpers.  This is a common piece for most of our processes. We want to be able to validate it, resolve refs, and build http servers from it.